### PR TITLE
Add the actual question and hint display

### PR DESCRIPTION
**Pull request by YacineSteeve**
[screen-capture (1).webm](https://user-images.githubusercontent.com/64146788/206586175-c6f132a7-7dda-40a2-9928-fbff8320c56f.webm)

### CODE REVIEW

File: frontend/src/components/Question.vue [hunk 1]
```diff
@@ -60,6 +60,32 @@ const difficultiesColors: Record<Quizoot.Difficulty, string> = {
 const questionDifficultyColor: ComputedRef = computed(
     () => difficultiesColors[props.question.difficulty]
 );
+
+const hintIsDisplayed: Ref<boolean> = ref(false);
```
Comment (chrichri17) [inline frontend/src/components/Question.vue:64]: `displayHint` is better right ?

File: frontend/src/components/Question.vue [hunk 2]
```diff
@@ -93,7 +134,10 @@ const questionDifficultyColor: ComputedRef = computed(
             </navigation-button>
             <navigation-button
                 v-if="!props.questionsFlow.isLastQuestion"
-                @click="goToNextQuestion"
+                @click="
+                    resetHint();
```
Comment (chrichri17) [inline frontend/src/components/Question.vue]: Is this really necessary? If I've already displayed the hint, whether I go back or forth doesn't matter

**Review by chrichri17 — APPROVED** [review]
lgtm

**Comment by chrichri17** [comment]
Just a heads-up: after viewing the video, I believe the buttons are not well aligned with the question content. You should fix that

**Comment by YacineSteeve** [comment]
Its because the buttons are centered but the question content is aligned to left. Should it be centered instead @chrichri17  ?

File: frontend/src/components/Question.vue (hunk 1 above)
Comment (YacineSteeve) [reply frontend/src/components/Question.vue:64]: yeah

File: frontend/src/components/Question.vue (hunk 2 above)
Comment (YacineSteeve) [reply frontend/src/components/Question.vue]: Well seen... And as the user can hide again the hint himself... Okay let me change this.

**Comment by chrichri17** [comment]
> Its because the buttons are centered but the question content is aligned to left. Should it be centered instead @chrichri17  ?

Nope.! The text alignment is fine. Maybe change the margin or the padding... ?

**Comment by YacineSteeve** [comment]
Looks better ?

Screenshot from 2022-12-09 15-49-31

**Comment by chrichri17** [comment]
YES

**Pull request merged**